packages/js/client-config-builder/package.json

```diff
@@ -24,20 +24,17 @@
   },
   "dependencies": {
     "@polywrap/core-js": "0.10.0-pre.8",
-    "@polywrap/ens-resolver-plugin-js": "0.10.0-pre.8",
-    "ethereum-provider-js": "0.1.7",
     "@polywrap/fs-plugin-js": "~0.10.0-pre",
-    "@polywrap/fs-resolver-plugin-js": "0.10.0-pre.8",
     "@polywrap/http-plugin-js": "~0.10.0-pre",
-    "@polywrap/http-resolver-plugin-js": "0.10.0-pre.8",
     "@polywrap/ipfs-plugin-js": "~0.10.0-pre",
-    "@polywrap/ipfs-resolver-plugin-js": "~0.10.0-pre",
     "@polywrap/logger-plugin-js": "0.10.0",
+    "@polywrap/plugin-js": "0.10.0-pre.8",
     "@polywrap/uri-resolver-extensions-js": "0.10.0-pre.8",
     "@polywrap/uri-resolvers-js": "0.10.0-pre.8",
     "@polywrap/wasm-js": "0.10.0-pre.8",
     "base64-to-uint8array": "1.0.0",
-    "concurrent-plugin-js": "0.1.2"
+    "concurrent-plugin-js": "0.1.2",
```

Comment:
We also need to publish concurrent-plugin under polywrap.

Reply:
Will fix this in another PR, thank you for catching this!